From a976a45a2047a840e959f27119290500dc751420 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Mon, 4 Nov 2019 10:44:49 +0800
Subject: [PATCH] dt-bindings: usb-xhci: add usb3-dis-autosuspend property

This patch adds a new "usb3-dis-autosuspend" property for some
Rockchip platforms which don't support autosuspend well.

Change-Id: I4d035415456405d47b85b3619fb47eb617fd2303
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 Documentation/devicetree/bindings/usb/usb-xhci.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/usb/usb-xhci.txt b/Documentation/devicetree/bindings/usb/usb-xhci.txt
index d520675ac44a..58a59299eb11 100644
--- a/Documentation/devicetree/bindings/usb/usb-xhci.txt
+++ b/Documentation/devicetree/bindings/usb/usb-xhci.txt
@@ -35,6 +35,7 @@ Optional properties:
     second one
   - usb2-lpm-disable: indicate if we don't want to enable USB2 HW LPM
   - usb3-lpm-capable: determines if platform is USB3 LPM capable
+  - usb3-dis-autosuspend: determines if the xHC supports USB3 autosuspend capable
   - quirk-broken-port-ped: set if the controller has broken port disable mechanism
   - xhci-slow-suspend: set if it needs an extraordinary delay to wait
     for xHC enter the Halted state after the Run/Stop (R/S) bit is cleared to '0'
-- 
2.35.3

